### PR TITLE
doc(lever): dbt source README edit for Fivetran style

### PR DESCRIPTION
**Pull request by fivetran-chloe**
Part of https://fivetran.atlassian.net/browse/RD-59826

Edits for Fivetran style. Rewords for clarity. **Please address my question (inline) and check my edits for technical accuracy.**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,7 +66,6 @@ Please create issues or open PRs against `master`. Check out [this post](https:/
 
 ## Database Support
 This package has been tested on BigQuery, Snowflake and Redshift.
-Coming soon -- compatibility with Spark!
```
Comment (fivetran-chloe) [inline README.md:72]: We do not advertise future changes - we only document them when they are live.

File: README.md [hunk 2]
```diff
@@ -1,18 +1,15 @@
...
+This package models Lever data from [Fivetran's Opportunity endpoint connector](https://fivetran.com/docs/applications/lever). It uses data in the format described by [this ERD](https://fivetran.com/docs/applications/lever#schemainformation).
+> NOTE: If your Lever connector was created [prior to July 2020](https://fivetran.com/docs/applications/lever/changelog) or still uses the Candidate endpoint, you must fully re-sync your connector or set up a new connector to use Fivetran's Lever dbt packages.
 
-This package aims to help you understand trends in recruiting, interviewing, and hiring at your company, as well as empower recruiting stakeholders with all potentially relevant information about individual opportunities, interviews, and jobs. It achieves this by:
+This package helps you understand trends in recruiting, interviewing, and hiring at your company. It also provides recruiting stakeholders with information about individual opportunities, interviews, and jobs. It achieves this by:
```
Comment (fivetran-chloe) [inline README.md:6]: The model package contains this exact content. Why is it in both places? Typically, the model package contains the package description and the source package describes exactly how the package manipulates your data (adds column-level testing, adds descriptions to tables, etc.) See the [Pardot source README](https://hub.getdbt.com/fivetran/pardot_source/latest/ ) for an example.

Comment (fivetran-jamie) [reply README.md:6]: good point -- i'll use the language from the pardot readme!

Comment (fivetran-chloe) [reply README.md:6]: @fivetran-jamie I don't see those changes in this PR - are you doing them in a separate PR?

Comment (fivetran-jamie) [reply README.md:6]: Ah apologies, yeah I just did them in master after merging this. here's that [commit](https://github.com/fivetran/dbt_lever_source/commit/5aaa186f44dde2e802a3d8610ee8e5c83dcc3b88)

Comment (fivetran-chloe) [reply README.md:6]: Thank you!

**Pull request merged**